clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala

```diff
@@ -22,13 +22,19 @@ trait ClioParsers {
   implicit def enumEntryParser[T <: EnumEntry: Enum](
     implicit c: ClassTag[T]
   ): ArgParser[T] = {
-    ArgParser.instance[T]("entry") { entry =>
-      val value = implicitly[Enum[T]].withNameOption(entry)
-      Either.cond(
-        value.isDefined,
-        value.get,
-        s"Unknown enum value $entry for type ${c.runtimeClass.getName} "
-      )
+    val enum = implicitly[Enum[T]]
+    val enumName = c.runtimeClass.getName
+    val entryMap = enum.namesToValuesMap
+    val entryString = entryMap.keys.mkString("|")
+
+    ArgParser.instance[T](entryString) { maybeEntry =>
```

Comment:
What does the argument `entryString` do here?

Reply:
It *should* be used to determine the value shown on `--help`, but it's broken in caseapp until alexarchambault/case-app#75 is merged and released, so I'm just doing it preemptively here.